src/EnergyPlus/DataHVACGlobals.hh

```diff
@@ -444,6 +444,7 @@ struct HVACGlobalsData : BaseGlobalStruct
     Real64 ElecHeatingCoilPower = 0.0;     // Electric power consumed by electric heating coil
     Real64 SuppHeatingCoilPower = 0.0;     // Electric power consumed by electric supplemental heating coil
     Real64 AirToAirHXElecPower = 0.0;      // Electric power consumed by Heat Exchanger:Air To Air (Generic or Flat Plate)
+    Real64 DefrostElecPower = 0.0;         // Electric power consumed by DX heating coil for defrosting (Resistive or ReverseCycle)
```

Comment:
Added new global variable for defrost electric power.

Reply:
Technically not a "global" variables (we're not supposed to use those anymore), but I know what you mean.